oak_functions_service/benches/wasm_benchmark.rs

```diff
@@ -66,7 +65,9 @@ fn bench_invoke_lookup(c: &mut Criterion) {
     const KEY_INDEX: i32 = 100;
 
     let test_data = create_test_data(0, MAX_DATA_SIZE);
-    test_state.lookup_data_manager.extend_next_lookup_data(test_data.clone());
+    test_state
+        .lookup_data_manager
+        .extend_next_lookup_data(test_data.iter().map(|(k, v)| (&k[..], &v[..])));
```

Comment:
Is the `&v[..]` syntax actually needed? Is there an equivalent method to call more explicitly (e.g. `as_ref` or something along those lines)?

Reply:
Yep, `as_ref` works.